lib/ansible/executor/task_executor.py

```diff
@@ -758,9 +758,14 @@ def _get_action_handler(self, connection, templar):
         Returns the correct action plugin to handle the requestion task action
         '''
 
+        network_group_modules = frozenset(['eos', 'nxos', 'ios', 'iosxr', 'junos', 'vyos'])
```

Comment:
Eek, having this list hardcoded in the task_executor feels wrong... I'm trying to think of a sane way to do it that wouldn't require actually loading all the action plugins to have them register an intent like this (and that also wouldn't open up the potential for "catchall" action plugins that could hijack things).